test/fixtures/dialects/db2fori/.sqlfluff

```diff
@@ -0,0 +1,2 @@
+[sqlfluff]
+dialect = sqlite
```

Comment:
This is wrong (though not used for generation, but helpful when you parse a file manually):

```suggestion
dialect = db2fori
```

Reply:
https://github.com/sqlfluff/sqlfluff/pull/3231/commits/b2477587ca57b2dd106d62823827683abd93dbeb thanks!